src/content/docs/architecture.mdx

```diff
@@ -139,51 +141,53 @@ those changes required forking Flutter.
 
 We had to fork 4 Flutter and Dart repositories to make Shorebird work:
 
+#### `flutter/buildroot` "The Buildroot"
+
+The `buildroot` repo contains the build scripts that are used to build the
+Flutter engine for various platforms. It's separate from `flutter/engine` in
+order to share code and configuration with the Fuchsia build system.
+
+We forked this code to make one small modification to the build setup to
+allow exposing our updater symbols from the Flutter engine up to
+`package:shorebird_code_push` and to include `libunwind` since the Rust code
+requires it for backtraces.
```

Comment:
```suggestion
We have made very few changes to this repo:

1. Expose our updater symbols from the Flutter engine to `package:shorebird_code_push`
2. Include `libunwind` since the Rust code requires it for backtraces.
```